libsoftwarecontainer/src/gateway/dbus/dbusgatewayinstance.cpp

```diff
@@ -123,20 +134,26 @@ bool DBusGatewayInstance::activateGateway()
     return testDBusConnection(config);
 }
 
-bool DBusGatewayInstance::testDBusConnection(const std::string &config)
+bool DBusGatewayInstance::testDBusConnection(const std::string &cfg)
```

Comment:
I'm not super keen on it either. But I wanted the variable that's actually used to have the best name possible in the method. The "cfg" variable is modified and then named "config", which is then what's used in the rest of the code. Do you have an alternative that you thing would work better for this case?